addons/website/static/src/components/configurator/configurator.js

```diff
@@ -617,6 +618,30 @@ async function applyConfigurator(self, themeName) {
         self.env.router.navigate({to: 'CONFIGURATOR_PALETTE_SELECTION_SCREEN'});
         return;
     }
+
+    async function attemptConfiguratorApply(data, retryCount = 0) {
+        try {
+            const resp = await self.rpc({
+                model: 'website',
+                method: 'configurator_apply',
+                kwargs: data,
+            });
+            window.sessionStorage.removeItem(SESSION_STORAGE_ITEM_NAME);
+            window.location = resp.url;
+        } catch (error) {
+            if (retryCount < 3) {
+                await concurrency.delay(5000);
+                return attemptConfiguratorApply(data, retryCount + 1);
+            }
+            // Wait a bit before allowing manual retry.
+            window.setTimeout(() => {
+                const loaderEl = document.querySelector('.o_theme_install_loader_container');
+                loaderEl.parentElement.removeChild(loaderEl);
```

Comment:
```suggestion
                document.querySelector('.o_theme_install_loader_container').remove();
```

Reply:
Yep, some time ago we avoided `remove` because not supported at all on IE, but we don't care about IE since a while now :wink: (especially in this "backend" screen)